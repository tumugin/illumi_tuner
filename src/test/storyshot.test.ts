import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  framework: 'vue',
})
