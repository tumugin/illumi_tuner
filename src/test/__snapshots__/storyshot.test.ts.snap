// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots UI/CharacterItem Basic 1`] = `
<div
  class="characterBox"
  style="background-color: rgb(234, 91, 118);"
>
  <div
    class="textContainer"
  >
    <div
      class="titleView whiteBox"
    >
      
      STAR ELEMENTS
    
    </div>
     
    <div
      class="whiteBox characterNameView"
    >
      
      天羽光駆
    
    </div>
  </div>
</div>
`;

exports[`Storyshots UI/LiveSelectView Basic 1`] = `
<div>
  <div
    class="dropdown b-dropdown btn-group"
  >
    <!---->
    <button
      aria-expanded="false"
      aria-haspopup="true"
      class="btn dropdown-toggle btn-secondary"
      type="button"
    >
      公演から選択する
    </button>
    <ul
      class="dropdown-menu"
      role="menu"
      tabindex="-1"
    >
      <!---->
       
      <div
        class="live-list-desktop"
      >
        <li
          role="presentation"
        >
          <a
            class="dropdown-item"
            href="#"
            role="menuitem"
            target="_self"
          >
            <span
              class="badge badge-primary"
            >
              
            2019/01/01
          
            </span>
             
            <span
              class="badge badge-dark"
            >
              
            さいたまスーパーアリーナ
          
            </span>
             
            <div>
              
            THE IDOLM@STER MILLION LIVE! 俺が考えた最強のライブ starring 俺
          
            </div>
          </a>
        </li>
        <li
          role="presentation"
        >
          <a
            class="dropdown-item"
            href="#"
            role="menuitem"
            target="_self"
          >
            <span
              class="badge badge-primary"
            >
              
            2019/02/01
          
            </span>
             
            <span
              class="badge badge-dark"
            >
              
            メットライフドーム
          
            </span>
             
            <div>
              
            THE IDOLM@STER MILLION LIVE! 神すぎて神になるライブ
          
            </div>
          </a>
        </li>
        <li
          role="presentation"
        >
          <a
            class="dropdown-item"
            href="#"
            role="menuitem"
            target="_self"
          >
            <span
              class="badge badge-primary"
            >
              
            2019/03/01
          
            </span>
             
            <span
              class="badge badge-dark"
            >
              
            東京ドーム
          
            </span>
             
            <div>
              
            THE IDOLM@STER MILLION LIVE! 7th 東京ドーム公演
          
            </div>
          </a>
        </li>
        <li
          role="presentation"
        >
          <a
            class="dropdown-item"
            href="#"
            role="menuitem"
            target="_self"
          >
            <span
              class="badge badge-primary"
            >
              
            2019/04/01
          
            </span>
             
            <span
              class="badge badge-dark"
            >
              
            日産スタジアム
          
            </span>
             
            <div>
              
            THE IDOLM@STER ALL PRODUCTION ぶち上がり合同ライブ
          
            </div>
          </a>
        </li>
      </div>
    </ul>
  </div>
   
  <!---->
</div>
`;

exports[`Storyshots UI/LoadingView Basic 1`] = `<!---->`;

exports[`Storyshots UI/PenlightItem Basic 1`] = `
<div
  class="container"
  style="background-color: rgb(58, 117, 187);"
>
  <span
    class="title"
  >
    ほげほげの色
  </span>
   
  <div
    class="idolListContainer"
  >
    <span
      class="idol"
    >
      <div
        class="colorIndicator"
        style="background-color: rgb(234, 91, 118);"
      />
      
      天羽光駆
    
    </span>
    <span
      class="idol"
    >
      <div
        class="colorIndicator"
        style="background-color: rgb(245, 173, 59);"
      />
      
      神崎水桜
    
    </span>
    <span
      class="idol"
    >
      <div
        class="colorIndicator"
        style="background-color: rgb(146, 207, 187);"
      />
      
      草薙星蘭
    
    </span>
  </div>
</div>
`;
